components/library/spidermonkey/Makefile

```diff
@@ -18,7 +18,7 @@ COMPONENT_NAME=	          spidermonkey
 COMPONENT_VERSION=        1.8.5
 COMPONENT_SUMMARY=        Javascript engine
 COMPONENT_PROJECT_URL=    https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey
-COMPONENT_FMRI=	          runtime/spidermonkey
+COMPONENT_FMRI=	          library/javascript/spidermonkey
```

Comment:
as I understand, Oracle took library/javascript/* for JS libraries. Let's use just library/spidermonkey, as it's C library for JS processing :)